arm/Microsoft.ServiceBus/namespaces/.bicep/nested_privateEndpoints.bicep

```diff
@@ -35,7 +35,7 @@ resource privateEndpoint_name 'Microsoft.Network/privateEndpoints@2021-05-01' =
     customDnsConfigs: privateEndpoint_var.customDnsConfigs
   }
 
-  resource privateEndpoint_name_default 'privateDnsZoneGroups@2020-05-01' = if (!empty(privateEndpoint_var.privateDnsZoneResourceIds)) {
+  resource privateEndpoint_name_default 'privateDnsZoneGroups@2017-04-01' = if (!empty(privateEndpoint_var.privateDnsZoneResourceIds)) {
```

Comment:
Was that intended?

Reply:
No. Will fix it

Reply:
Just saw you also updated this one to a newer one. Please also test

Reply:
Then again - the parameter file doesn't test it anyways (though it should)